src/arcaflow_plugin_sdk/schema.py

```diff
@@ -5621,7 +5587,11 @@ def __init__(
     ):
         # noinspection PyArgumentList
         OneOfStringSchema.__init__(
-            self, types, discriminator_inlined, discriminator_field_name
+            self,
+            types=types,
+            discriminator_inlined=discriminator_inlined,
+            discriminator_field_name=discriminator_field_name,
+            # oneof_type="_discriminated_string_",
         )
```

Comment:
you're guess is as good as mine

Reply:
i removed the commented code

Reply:
If you remove the directive, the linter doesn't tell you why it objects?

Reply:
nope; it seems like pycharm should object b/c we're accessing a private method

Reply:
Did you reply to the right comment, here?  Or, did you mean to respond to [the other one](https://github.com/arcalot/arcaflow-plugin-sdk-python/pull/131#discussion_r1583482978)?  (I had resolved it, but I've unresolved it now, for your convenience.)

Reply:
Yeah I don't know about the necessity of this PyCharm directive, but every *Schema dataclass appears to have it, so I believe addressing it in this pr would be out of scope.

Reply:
I agree, but I think it needs to be addressed in a separate issue.

Reply:
😢  In this PR you modified this call (not to mention the thing being called)...so it's possible that this linter disable is no longer needed.  And, in concert with the other changes you've made in the past two days, it would have been easy to find out what the objection is that we're disabling here (if there still is one) and to add the annotation that I asked for.

Instead, we'll incur another small bit of technical debt.  😖